.buildkite/pipeline.yml

```
@@ -30,6 +30,13 @@ steps:
   #################
   # Lint
   #################
+  - label: ":rust: Lint"
+    command: |
+      # echo "--- :rust: Running Clippy"
+      # make lint-rust
+
+      echo "--- :rust: Ensuring Code Conforms to rustfmt"
+      make fmt-check-rust
```

Probably worth adding a `notify` so that the status is also reported in GitHub PRs:

```suggestion
  - label: ":rust: Lint"
    command: |
      # echo "--- :rust: Running Clippy"
      # make lint-rust

      echo "--- :rust: Ensuring Code Conforms to rustfmt"
      make fmt-check-rust
    notify:
      - github_commit_status:
          context: "Rust Lint"
```

---

Addressed [here](https://github.com/Automattic/fluent-rust-tools/pull/6#discussion_r2181057143).